packages/donor/src/screens/extendedSignup/PersonalDetails.tsx

```diff
@@ -55,15 +57,34 @@ export default function PersonalDetails(props: PersonalDetailsProps) {
     erorrMessage = "מספר הטלפון אינו תקין";
   }
 
-  const areAllFieldsValid = !lastNameError && isValidPhone && bloodType;
+  const areAllFieldsValid =
+    !firstNameError && !lastNameError && isValidPhone && bloodType;
+  const allFieldsHaveValue =
+    !isEmpty(firstName) && !isEmpty(lastName) && !isEmpty(phone);
+
+  const setFirstNameAndValidate = (newFirstName: string) => {
+    let firstNameErrorMessage = "";
+
+    if (newFirstName.length < 2) {
+      firstNameErrorMessage = "נא להזין שם פרטי";
```

Comment:
I think the message should be something like ״שם אינו תקין״

What do you think?

Reply:
Sure :)